src/extensions/scratch3_boost/index.js

```diff
@@ -101,18 +101,30 @@ const BoostPort = {
 };
 
 /**
- * Enum for indexed colors read by the Boost Vision Sensor
+ * Ids for each color sensed by the Boost Vision Sensor
+ */
+const COLOR_ID_ANY = 'any';
```

Comment:
I tried to simplify this by not needing the extra color ID values and wasn't successful at finding a better alternative.  However, I wonder if these could get renamed to `COLOR_ANY` and `COLOR_RED` etc, for a bit of simplification?